### PR TITLE
 Define behavior for all bitsPerSecond attributes

### LINKED ISSUE #169: What are the default bitrate attribute values?
The [videoBitsPerSecond](https://w3c.github.io/mediacapture-record/MediaRecorder.html#dom-mediarecorder-videobitspersecond) read-only attribute says *"The value of the Video encoding target bit rate that was passed to the Platform (potentially truncated, rounded, etc), or the calculated one if the user has specified bitsPerSecond."*

This does not say what to return If neither `videoBitsPerSecond` nor `bitsPerSecond` were passed. Ditto for [audioBitsPerSecond](https://w3c.github.io/mediacapture-record/MediaRecorder.html#dom-mediarecorder-audiobitspersecond).

Possible solutions:
 1. return `0`
 2. Make `audioBitsPerSecond` and `videoBitsPerSecond` attributes nullable.

### PR DESCRIPTION
**Pull request by Pehrsons**
Fixes #169.

### CODE REVIEW

**Comment by Pehrsons** [comment]
Note that the first commit in this PR is a squash of PR #171, so consider it a dependency.

Find the diff to be reviewed at https://github.com/w3c/mediacapture-record/pull/175/commits/6112e298f76e1f19d8e71206942be7e926d88da6.

File: MediaRecorder.bs [hunk 1]
```diff
@@ -248,6 +262,27 @@ interface MediaRecorder : EventTarget {
...
 
+    <li>If |recorder|'s [=[[ConstrainedBitsPerSecond]]=] slot is not
+    <code>undefined</code>, set |recorder|'s {{videoBitsPerSecond}} and
+    {{audioBitsPerSecond}} attributes to values the User Agent deems reasonable
+    for the respective media types, for recording all tracks in |tracks|, such
+    that the sum of {{videoBitsPerSecond}} and {{audioBitsPerSecond}} is close
+    to the value of |recorder|'s [=[[ConstrainedBitsPerSecond]]=] slot.</li>
+
+    <li>Let |videoBitrate| be the value of |recorder|'s {{videoBitsPerSecond}}
+    attribute, and constrain the configuration of |recorder| to target a bitrate
```
Comment (jan-ivar) [inline MediaRecorder.bs]: How about: "to target an aggregate bitrate"?

File: MediaRecorder.bs [hunk 2]
```diff
@@ -248,6 +262,27 @@ interface MediaRecorder : EventTarget {
...
+    of |videoBitrate| bits per second for all video tracks |recorder| will be
+    recording. |videoBitrate| is a hint for the encoder and the value might be
+    surpassed, not achieved, or only be achieved over a long period of
+    time.</li>
+
+    <li>Let |audioBitrate| be the value of |recorder|'s {{audioBitsPerSecond}}
+    attribute, and constrain the configuration of |recorder| to target a bitrate
```
Comment (jan-ivar) [inline MediaRecorder.bs]: Ditto.

**Review by jan-ivar — APPROVED** [review]
Lgtm!

**Comment by jan-ivar** [comment]
@armax00 PTAL

**Comment by armax00** [comment]
lgtm for me too. I'll file a bug on Chromium for fixing the behavior, thanks @jan-ivar for checking.

**Comment by Pehrsons** [comment]
Rebased. ping @jan-ivar

**Pull request merged**